dataplane/standalone/pkthandler/main.go

```diff
@@ -73,16 +74,26 @@ func main() {
 		log.Exit(err)
 	}
 
-	var wg sync.WaitGroup
-	wg.Add(2)
+	errCh := make(chan error)
 	go func() {
-		h.ManagePorts(portCtl)
-		wg.Done()
+		if err := h.ManagePorts(portCtl); err != nil {
+			errCh <- fmt.Errorf("HostPortControl rpc exited with err: %v", err)
+			return
+		}
+		errCh <- fmt.Errorf("HostPortControl rpc exited without error")
 	}()
 	go func() {
-		h.StreamPackets(packet)
-		wg.Done()
+		if err := h.StreamPackets(packet); err != nil {
+			errCh <- fmt.Errorf("StreamPackets rpc exited with err: %v", err)
+			return
+		}
+		errCh <- fmt.Errorf("StreamPackets rpc exited without error")
 	}()
 
-	wg.Wait()
+	err = <-errCh
```

Comment:
With this we only get one of the message?

Reply:
this is intentional, if either rpc's exit we need to exit